drivers/flash/flash_stm32_qspi.c

```diff
@@ -29,19 +30,21 @@
 
 #define STM32_QSPI_RESET_GPIO DT_INST_NODE_HAS_PROP(0, reset_gpios)
 #if STM32_QSPI_RESET_GPIO
-#include <drivers/gpio.h>
+#include <zephyr/drivers/gpio.h>
 #endif
```

Comment:
While you're at it, I'd find it reasonnable to remove the conditional around this header inclusion